frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -814,6 +881,15 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
               )}
             </p>
           </Alert>
+        )}{' '}
+        {tokenizedAuth === 'GCP' && showSTSWarn && (
+          <STSWarningAlert
+            title={t('olm~Cluster in GCP Workload Identity / Federated Identity Mode')}
+            message={t(
+              'olm~This cluster is using GCP Workload Identity / Federated Identity to reach the cloud API. In order for this operator to take the actions it requires directly with the cloud API, provide the Client ID, Tenant ID, and Subscription ID during installation. Manual subscriptions are highly recommended as steps should be taken before upgrade to ensure that the permissions required by the next version are properly accounted for in the role. See the operator description for more details.',
```

Comment:
The GCP WIF will have different variable names:

Project ID
Pool ID
Provider ID
Service Account Email